test/fixtures/vcr/relay_group_add_relay.json

```diff
@@ -11,14 +11,14 @@
       "url": "http://localhost:4000/v1/token"
     },
     "response": {
-      "body": "{\"token\":{\"value\":\"w05n5iZL4peQoZ1gibP1MR1TgvSTXrIwZzCIJYRjR+0=\"}}",
```

Comment:
I'm not sure I understand @christophermaier. I'm with you on keeping the PR focused, but how do I exclude the vcr bits and have the tests pass? I thought you had to rerecord the cassette whenever the test changed.